packages/facility-server/app/sync/FacilitySyncManager.js

```diff
@@ -194,15 +197,18 @@ export class FacilitySyncManager {
     log.info('FacilitySyncManager.snapshottingOutgoingChanges', { pushSince });
     const modelsForPush = getModelsForPush(this.models);
     const outgoingChanges = await snapshotOutgoingChanges(this.sequelize, modelsForPush, pushSince);
-    if (outgoingChanges.length > 0) {
+    const outgoingChangesWithChangelogs = await attachChangelogToSnapshotRecords(this.sequelize, outgoingChanges, {
+      minSourceTick: pushSince,
+    });
+    if (outgoingChangesWithChangelogs.length > 0) {
       log.info('FacilitySyncManager.pushingOutgoingChanges', {
-        totalPushing: outgoingChanges.length,
+        totalPushing: outgoingChangesWithChangelogs.length,
       });
       if (this.__testSpyEnabled) {
-        this.__testOnlyPushChangesSpy.push({ sessionId, outgoingChanges });
+        this.__testOnlyPushChangesSpy.push({ sessionId, outgoingChanges: outgoingChangesWithChangelogs });
       }
-      await pushOutgoingChanges(this.centralServer, sessionId, outgoingChanges);
-      await deleteRedundantLocalCopies(modelsForPush, outgoingChanges);
+      await pushOutgoingChanges(this.centralServer, sessionId, outgoingChangesWithChangelogs);
+      await deleteRedundantLocalCopies(modelsForPush, outgoingChangesWithChangelogs);
```

Comment:
```suggestion
      await deleteRedundantLocalCopies(modelsForPush, outgoingChanges);
```
Don't think it needs changelogs right?